components/moderation-panel/ModEditSubmissionForm.vue

```diff
@@ -513,11 +616,23 @@ const hasHealthcareProfessionalChanges = (submissionBeforeChangesComparison: Sub
             submissionBeforeChangesComparison?.healthcareProfessionals?.[0]?.facilityIds || [],
             hpSectionFields.facilityIds
         )
+        || ((submissionBeforeChangesComparison?.healthcareProfessionals?.[0]?.additionalInfoForPatients ?? '')
+          !== (hpSectionFields.additionalInfoForPatients ?? ''))
     )
 }
 
-const formHasUnsavedChanges = () => hasFacilityChanges(submissionBeforeChanges.value)
-  || hasHealthcareProfessionalChanges(submissionBeforeChanges.value)
+const formHasUnsavedChanges = () => {
+    const initialSubmission = submissionBeforeChanges.value
+    if (!initialSubmission) {
+        return false
+    }
+
+    const hasNotesChanges = (initialSubmission.notes ?? '') !== currentSubmissionNotes.value
+    const facilityChanges = hasFacilityChanges(initialSubmission)
+    const hpChanges = hasHealthcareProfessionalChanges(initialSubmission)
+
+    return facilityChanges || hpChanges || hasNotesChanges
```

Comment:
tip: I recommend using return variables! So you make this a variable like `hasChanges`, then return hasChanges. 
These make it easier to understand the check PLUS you can now debug and check the variable easily